jest-preset.front.js

```diff
@@ -30,6 +30,8 @@ const moduleNameMapper = {
 module.exports = {
   rootDir: __dirname,
   moduleNameMapper,
+  /* Tells jest to ignore duplicated manual mock files, such as index.js */
+  modulePathIgnorePatterns: ['.*__mocks__.*'],
```

Comment:
@joshuaellis The internet recommends different solutions, but I think this is one that works best for us:

> An array of regexp pattern strings that are matched against all module paths before those paths are to be considered 'visible' to the module loader. If a given module's path matches any of the patterns, it will not be require()-able in the test environment.

https://jestjs.io/docs/configuration#modulepathignorepatterns-arraystring

Reply:
I don't think it needs more discussion, let's just move forward with it :)